### PR TITLE
Jashton/instance names

### PR DESCRIPTION
**Pull request by JeffAshton**

### CODE REVIEW

File: src/org/OrgRulesMapper.js [hunk 1]
```diff
@@ -29,6 +29,38 @@ function* mapClauses( definition, instanceCatalog ) {
...
+				duplicatesDeep( [
+					instanceNames
+				] )
+			);
+
+			const duplicatesStr = _.join( duplicates, ', ' );
+			throw new Error( `Instance names are duplicated in rule: ${duplicatesStr}` );
+		}
+
+		yield {
```
Comment (JeffAshton) [inline src/org/OrgRulesMapper.js:56]: Attribute defined in LMS here:

https://search.d2l.dev/xref/lms/lp/framework/core/D2L.LP/LP/LaunchDarkly/Default/LaunchDarklyUserProvider.cs?r=0ccdf8d6#47

File: schemas/org-boolean/v4_3.json [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+							"ca3d",
+							"ca3p",
+							"eu01",
+							"us04",
+							"dmz",
+							"devlms"
+						]
+					}
+				},
+				"instanceNames": {
```
Comment (JeffAshton) [inline schemas/org-boolean/v4_3.json:190]: Adding `instanceNames` which allows you to enable an org flag for an entire instance.

File: schemas/org-multivariate/v4_3.json [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+							"ca3d",
+							"ca3p",
+							"eu01",
+							"us04",
+							"dmz",
+							"devlms"
+						]
+					}
+				},
+				"instanceNames": {
```
Comment (JeffAshton) [inline schemas/org-multivariate/v4_3.json:179]: Adding `instanceNames` which allows you to enable an org flag for an entire instance.

File: schemas/org-boolean/v4_3.json [hunk 4]
```diff
@@ -0,0 +1,395 @@
+{
```
Comment (JeffAshton) [inline schemas/org-boolean/v4_3.json:1]:
```diff
$ diff v4_2.json v4_3.json 
3c3
< 	"$id": "http://schemas.featureflags.d2ldev.com/lms/org-boolean/v4_2.json",
---
> 	"$id": "http://schemas.featureflags.d2ldev.com/lms/org-boolean/v4_3.json",
17c17
< 				"http://schemas.featureflags.d2ldev.com/lms/org-boolean/v4_2.json"
---
> 				"http://schemas.featureflags.d2ldev.com/lms/org-boolean/v4_3.json"
187a188,199
> 					}
> 				},
> 				"instanceNames": {
> 					"type": "array",
> 					"title": "The set of instance names to target",
> 					"additionalItems": false,
> 					"minItems": 1,
> 					"uniqueItems": true,
> 					"items": {
> 						"type": "string",
> 						"title": "The instance names to target",
> 						"minLength": 1
```

File: schemas/org-multivariate/v4_3.json [hunk 5]
```diff
@@ -0,0 +1,395 @@
+{
```
Comment (JeffAshton) [inline schemas/org-multivariate/v4_3.json:1]:
```
$ diff v4_2.json v4_3.json 
```
```diff
3c3
< 	"$id": "http://schemas.featureflags.d2ldev.com/lms/org-multivariate/v4_2.json",
---
> 	"$id": "http://schemas.featureflags.d2ldev.com/lms/org-multivariate/v4_3.json",
18c18
< 				"http://schemas.featureflags.d2ldev.com/lms/org-multivariate/v4_2.json"
---
> 				"http://schemas.featureflags.d2ldev.com/lms/org-multivariate/v4_3.json"
176a177,188
> 					}
> 				},
> 				"instanceNames": {
> 					"type": "array",
> 					"title": "The set of instance names to target",
> 					"additionalItems": false,
> 					"minItems": 1,
> 					"uniqueItems": true,
> 					"items": {
> 						"type": "string",
> 						"title": "The instance names to target",
> 						"minLength": 1
```

**Review by j3parker — APPROVED** [review]

**Pull request merged**